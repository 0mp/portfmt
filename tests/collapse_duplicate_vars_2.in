MAKE_ARGS=	pkgconfigdir="${PREFIX}/libdata/pkgconfig"
MAKE_ARGS=	INSTALL_STRIP_FLAG="${STRIP}"
