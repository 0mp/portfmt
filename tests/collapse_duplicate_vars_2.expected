MAKE_ARGS=	INSTALL_STRIP_FLAG="${STRIP}" \
		pkgconfigdir="${PREFIX}/libdata/pkgconfig"
